met-web/src/components/engagement/admin/create/authoring/AuthoringSummary.tsx

```diff
@@ -42,25 +47,48 @@ const AuthoringSummary = () => {
         }
     }, [fetcher.data]);
 
-    const { content } = useRouteLoaderData('authoring-loader') as {
-        content: Promise<EngagementContent[]>;
+    const untouchedDefaultValues: EngagementUpdateData = {
```

Comment:
The defaultValues are the defaultValues for the form. They are passed down from the AuthoringContext and so is setDefaultValues, so that we have full control over the value. It represents the "untouched" state of the form, which will change based on the values that we load from the db. So the regular defaultValues are whatever is loaded into the form. The untouched default values are the blank default values that are never changed.